examples/bri-3/src/bri/zeroKnowledgeProof/services/blockchain/blockchain.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class BlockchainService {
+  async write(ccsmAnchorHash: string) {
```

Comment:
you can remove this or add lint ignore here if this will be used soon, but currently there is a lint warning